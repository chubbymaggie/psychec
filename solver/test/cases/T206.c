void k2() {
    j = j1 + j2;
}
