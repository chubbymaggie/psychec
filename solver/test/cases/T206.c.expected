/* Forward declarations */

/* Type definitions */

/* Variables and functions */
scalar_t__ j ;
scalar_t__ j1 ;
scalar_t__ j2 ;
